.github/workflows/docs-check.yml

```diff
@@ -14,18 +14,13 @@ jobs:
         uses: actions/checkout@v3
 
       - name: Install Python
-        uses: actions/setup-python@v1
+        uses: actions/setup-python@v5
```

Comment:
`v5` is the latest major version of [`actions/setup-python`](https://github.com/actions/setup-python) at the time of this writing.